index.html

```diff
@@ -0,0 +1,200 @@
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="far fa-comments"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    Forum
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Listen to speakers from various countries about the messeges of sharing and opening.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-praying-hands"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    Workshop
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Try creating your own work using open source license rather than just watching.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-project-diagram"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    DMC Ignite
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Get opporunites to network with DMC affiliates around the world, also after the summit.
+                                </p>
+                            </div>
+                        </li>
+                    </ul>
+                    <button class="btn btn-large" type="button">SEE THE WHOLE PROGRAM</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+              <div class="speakers">
+                  <h2 class="speakers-heading">Featured Speakers</h2>
+                    <ul class="speakers-list" id="speakers-list">
+                        
+                    </ul>
+               </div>
```

Comment:
- [ ] In the featured speakers section there is a background seen behind all images, kindly include it.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/128994257-e7c49575-24f2-4baf-8a92-f54b392599e4.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/128994303-ff7f76de-3664-40c4-9011-a324a535d11b.png' height='250px'> |